src/test/java/io/jenkins/plugins/analysis/warnings/ParsersITest.java

```diff
@@ -418,6 +418,12 @@ public void shouldProvideMessagesAndDescriptionForSecurityIssuesWithSpotBugs() {
                 "java/nio/file/Paths.get(Ljava/lang/String;[Ljava/lang/String;)Ljava/nio/file/Path; reads a file whose location might be specified by user input");
     }
 
+    /** Runs the Clang-Analyzer parser on an output file that contains 3 issues. */
+    @Test
+    public void shouldFindAllClangAnalyzerIssues() {
+        shouldFindIssuesOfTool(6, new ClangAnalyzer(), "ClangAnalyzer.txt");
```

Comment:
I think you have a typo in the filename: 
- the file is: `ClangAnalzer.txt`
- the fileName is: `ClangAnalyzer.txt`

Reply:
Thanks. I've also found the problem in log. Wonder why it wasn't triggered on my machine.

Reply:
And also there is 3 issues in the test file...